dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -892,4 +893,37 @@ portalRouter.route('/gef/mandatory-criteria/version/:version').get(mandatoryCrit
  */
 portalRouter.route('/utilisation-reports').post(utilisationReportUpload.postUtilisationReportData);
 
+/**
+ * @openapi
+ * /utilisation-reports/:_id:
+ *   get:
+ *     summary: Get utilisation report with the specified MongoDB ID ('_id')
+ *     tags: [UtilisationReport]
+ *     description: Get utilisation report with the specified MongoDB ID ('_id')
+ *     parameters:
+ *       - in: path
+ *         name: _id
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: the MongoDB ID for the required report
+ *     responses:
+ *       200:
+ *         description: OK
+ *         content:
+ *           application/json:
+ *             schema:
+ *               allOf:
+ *                 - $ref: '#/definitions/UtilisationReport'
+ *                 - type: object
+ *                   properties:
+ *                     _id:
+ *                       example: 123456abc
+ *       400:
+ *         description: Bad request
+ *       404:
+ *         description: Not found
+ */
+portalRouter.route('utilisation-reports/:_id').get(getUtilisationReportController.getUtilisationReportById);
```

Comment:
from your other comments I think I was expecting the _id validation to happen here?